doc/xltdoc/user-manual/index.html

```diff
@@ -10,6 +10,16 @@ <h2><i class="fa fa-book"></i> User Manual</h2>
     <p class="lead">A comprehensive User Manual to learn Xceptance LoadTest in detail.</p>
 </div>
 
+<div class="alert alert-danger" role="alert">
+  <p>
+      <span class="fa fa-warning" aria-hidden="true"></span>
+      You are viewing the legacy version of the XLT documentation and the manual chapter might not be longer up-to-date. The latest version can be found at <a href="https://xltdoc.xceptance.com/load-testing/manual/">https://xltdoc.xceptance.com/load-testing/manual/</a>.
```

Comment:
> You are viewing the legacy version of the XLT documentation and the manual chapter might not be longer up-to-date.

You are viewing the legacy version of the XLT documentation and the user manual might not longer be up-to-date.